src/amen.coffee

```diff
@@ -52,7 +52,13 @@ print = ([description, result], indent="") ->
         if result == true
           description.green
         else if result.message? and result.message != ""
-          "#{description.red} (#{result.message?.red})"
+          if result.message != "Test timed out"
+            """
+            #{description.red} (#{result.message.red})
+            #{result.stack.red}
```

Comment:
The other way to do this is to dump the error stack _at the time we encounter the error_, in the `catch` block. That way we don't clutter the test output itself. Other than that, this is great.